### DISCUSSION TITLE
How to i assign ALT + TAB to cycle via windows to XBOX controller

### DISCUSSION THREAD

**Discussion by icescoop12** [AntiMicroX/antimicrox#874] [Q&A]
hi 
   How can I assign (if this is the best option) ALT + TAB to the xbox series Guide button ,i do it via advanced e/g add two key assignment's ,but it wont cycle between pc windows if i press it, how do i get it to cycle via pc windows
can i assign ALT to the guide button then assign TAB to another button like the A button(but will only operate as TAB as long as the guide(ALT) button is held(how can i do this?, is it possible?)
this is so i can focus between various overlayed windows in games whilst playing in VR
thanks

**Comment by BoomerangAide** [comment]
If Alt and Tab are on separate buttons, Alt button can be set on toggle, then you can press tab button to cycle around, then press Alt button again to release Alt.

edit: I tried but failed, to switch to another set so that the original function of A is preserved in the initial set, allow to keep alt+tab opened while in the next set, and use tab as the function of the A button in the next set, while finding a way to revert to the initial set (maybe by holding A button long enough).
I even caused my gamepad to switch between its xbox mode and custom mode, so maybe my hardware add difficulty?
